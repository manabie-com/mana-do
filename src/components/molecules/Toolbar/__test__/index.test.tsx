import React from 'react';
import {render} from '@testing-library/react';
import {MenuActive, Toolbar} from "../index";

describe('<Toolbar /> render', () => {
    it('should match snapshot', function () {
        const Element = render(<Toolbar callback={jest.fn()} menuActive={MenuActive.IMPORTANT}/>);
        expect(Element.container.firstChild).toMatchSnapshot();
    });
})