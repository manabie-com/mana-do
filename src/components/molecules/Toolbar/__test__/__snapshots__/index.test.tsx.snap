// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Toolbar /> render should match snapshot 1`] = `
<div
  class="WrapperItemSiderBar"
>
  <div
    class="false ItemSiderBar"
  >
    <i
      class="manabie-endless"
    />
    <span>
      All
    </span>
  </div>
</div>
`;
